full-backend-tests/src/test/java/org/graylog/plugins/views/ScriptingApiResourceIT.java

```diff
@@ -267,6 +271,71 @@ void testGetRequestAcii() {
         org.assertj.core.api.Assertions.assertThat(response).isEqualTo(expected.trim());
     }
 
+    @ContainerMatrixTest
+    void testCsvRender() {
+        final InputStream response = given()
+                .spec(requestSpec)
+                .header(new Header("Accept", MoreMediaTypes.TEXT_CSV))
+                .when()
+                .body("""
+                        {
+                          "group_by": [
+                            {
+                              "field": "facility"
+                            }
+                          ],
+                          "metrics": [
+                            {
+                              "function": "count",
+                              "field": "facility"
+                            }
+                          ]
+                        }
+                        """)
+                .post("/search/aggregate")
+                .then()
+                .log().ifStatusCodeMatches(not(200))
+                .statusCode(200)
+                .extract().body().asInputStream();
+
+        final CsvParser csvParser = new CsvParser(Csv.parseRfc4180());
+        final List<String[]> lines = csvParser.parseAll(response);
+
+
+
+        // headers
+        Assertions.assertArrayEquals(lines.get(0), new String[]{"Grouping", "Metric : count"});
```

Comment:
I'd consider replacing "Metric : count" with ResponseSchemaEntry.metric(...).name()

Reply:
IDK, I feel that's better to have hardcoded values in the test. If the name() method fails and delivers empty string, we won't catch that in the test.